### PR TITLE
deleted double asymptoticnotation pt-br file

### PR DESCRIPTION
**Pull request by fnscoder**
#2889 
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: fr-fr/.directory [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[Dolphin]
+Timestamp=2017,10,6,22,19,4
+Version=4
+ViewMode=2
```
Comment (prertik) [inline fr-fr/.directory]: What is this file? I don't think this file is present in original repo!

File: pt-br/asymptoticnotation-pt.html.markdown [hunk 2]
```diff
@@ -1,161 +0,0 @@
----
-category: Algorithms & Data Structures
-name: Asymptotic Notation
-contributors:
-    - ["Jake Prather", "http://github.com/JakeHP"]
-translators: 
-    - ["Carolina Knoll", "http://github.com/carolinaknoll"]
-lang: pt-br
----
```
Comment (prertik) [inline pt-br/asymptoticnotation-pt.html.markdown]: Are you sure this file is inferior or incorrect than another version?

Comment (fnscoder) [reply pt-br/asymptoticnotation-pt.html.markdown]: Yes, the two were translated by the same person. I deleted the old version and kept the new and corrected version.

File: fr-fr/.directory (hunk 1 above)
Comment (fnscoder) [reply fr-fr/.directory]: Sorry, this is a manjaro system file. I forgot to put it in .gitignore. It's done!

**Comment by prertik** [comment]
Can you please send a new pull request not committing like `delete .gitignore` or `delete .directory` files. It's bad practice to merge files, not in the context of the pull request. Just delete the not good version and send the pull request committing only that and no extra changes other than that.

**Pull request closed without merging**